website/content/vagrant-cloud/hcp-vagrant/migration-guide.mdx

```diff
@@ -0,0 +1,91 @@
+
+Those who take no action will have their boxes migrated as part of the site wide migration. The Vagrant team will maintain redirects to ensure users can expect all existing URLs and workflows to operate as usual after migration regardless of how boxes are migrated.
+
+## Prerequisites
+
+Before you continue with this guide, you should familiarize yourself with a few new resource management concepts in HCP and HCP Vagrant Registry.
+
+###### Registries
+
+The collection that holds a user’s boxes, the registry name is incorporated into the first half of resource names. For example, `hashicorp/atlas`.
+
+~> Note: An HCP registry is analogous to an Organization in Vagrant Cloud. A given HCP project may have an unlimited number of registries.
+
+##### HCP Organizations
+
+[HCP organizations](/hcp/docs/hcp/admin/orgs) are parent-level entities that can have up to ten projects. A single HCP account can be a member of multiple organizations, as an invited user, but can only create and own a single organization.
+
+##### HCP Projects
+
+[HCP projects](/hcp/docs/hcp/admin/orgs) allow organization owners to segment resources by team, environment, etc. Resources such as HCP Vagrant Registries, Hashicorp Virtual Private Networks (HPN). Each organization is assigned a `default-project` at creation and may have up to 10 projects.
+
+#### Create an HCP Vagrant Registry account
+
+To use the Vagrant Cloud migration tool, you must define an organization and project within an HCP account and be signed into an active session on that account.
+
+Your HCP account must use the email address associated with your Vagrant Cloud account. Check under `Settings → Profile` if you are unsure what email address you use for Vagrant Cloud.
+
+1. Sign up for a (free) [HCP account](https://portal.cloud.hashicorp.com/orgs/create), with your primary Vagrant Cloud organization email address. You must verify your account and sign into HCP.
+2. At first log-in, use the wizard to create an HCP organization.
+3. On creation of your organization, you will be directed to the dashboard for your `default-project`. You can create an additional project from the Projects page.
+4. Now that you have a project in an HCP Account that uses the same email address as your Vagrant Cloud account, you can begin the migration process.
+
+## Migrate to HCP Vagrant Registry
+
+#### 1. Sign in to both accounts and open the migration tool
+
+Once you have signed into both your HCP and Vagrant Cloud accounts, you can access the [migration tool](https://app.vagrantup.com/migration).
+
+To auto-populate the migration tool with a specific Vagrant Cloud `Organization` click on the `Migrate to HCP` button in the top right corner of your [settings](https://app.vagrantup.com/settings) page.
```

Comment:
I think this line can be dropped. The auto-selected organization thing is really only useful for the button in the settings.

Reply:
Sounds good, I also removed a reference to auto-population two lines below this to be consistent.